developers_chamber/ecs_utils.py

```diff
@@ -193,25 +193,64 @@ def start_service(cluster, service, count, region):
     except ecs_client.exceptions.ClusterNotFoundException:
         raise ClickException("Cluster not found: '{}'".format(cluster))
     except ecs_client.exceptions.ServiceNotFoundException:
-        raise ClickException("Service not found: '{}'".format(cluster))
+        raise ClickException("Service not found: '{}'".format(service))
     except ClientError as ex:
         raise ClickException(ex)
 
 
-def start_services(cluster, count, region):
+def start_services(cluster, services, count, region):
+    ecs_client = _get_ecs_client(region)
+
+    if count is not None and count < 1:
+        raise ClickException("Count must be greater than zero or 'None' to be determined from autoscaling targets.")
+
+    for service in services:
+        count_per_service = count
+        # service = service.split('/')[1]
+        if count_per_service is None:
+            try:
+                count_per_service = get_min_capacity_for_service(cluster=cluster, service=service, region=region)
+            except ClickException as ex:
+                raise ClickException(
+                    'Set explicit count or set up autoscaling with minimum capacity.\n{}'.format(ex)
+                )
+
+        LOGGER.info('Starting service: {}'.format(service))
+        try:
+            ecs_client.update_service(
+                cluster=cluster,
+                service=service,
+                desiredCount=count_per_service,
+            )
+        except ecs_client.exceptions.ClusterNotFoundException:
+            raise ClickException("Cluster not found: '{}'".format(cluster))
```

Comment:
Dvojité uvozovky normálně nepoužíváme - pokud potřebuješ další jednoduché zobrazit v textu (které by ti jinak ukončily řetězec), měly by jít oescapovat, aby se zobrazily.
Ale koukám, že uvozovky v těch exceptions za dvojtečkou používáš různě - někde je ten vkládaný text "ouvozovkován", jinde ne - to by možná chtělo sjednotit.

Reply:
Měl jsem to původně escapované ve stringu a to se lubošovi nelíbilo. Nicméně v CLI je běžnější používat jednoduché uvozovky, takže podle PEP8 je správně to prohodit a neescapovat. Vím, radši bych to taky měl všude stejně, už jsem o tom vedl diskusi

Reply:
Takhle je to správně.

Reply:
Takhle to je správně.

Reply:
OK, pokud je to úzus a shodli jste se na tom, pak klidně, nemmám problém.